MiniDumper/MiniDumper.cs

```diff
@@ -178,6 +180,28 @@ public void PrintDebugString(OUTPUT_DEBUG_STRING_INFO outputDbgStrInfo)
             }
         }
 
+        public void MemoryCommitThreshold(int commitThreshold)
+        {
+            System.Timers.Timer aTimer = new System.Timers.Timer();
```

Comment:
This timer might stop working at any time - it is eligible for disposal when this method finishes. Please make it a field of the class and move the initialization to the constructor. We should also have a nice name for it (eg. `pollingTimer`) as we will probably also use it to query performance counters.